jamventProto/jamctrllib/JamCtrlHw.h

```diff
@@ -0,0 +1,28 @@
+#ifndef __JAM_CTRL_HW_H__
+#define __JAM_CTRL_HW_H__
+
+#include "JamCtrlMgr.h"
+
+class JamCtrlHw : public JamCtrlMgr
```

Comment:
I have to get my thinking straight here.  I was thinking that this is the controller that managers the HW.  That is
not true.  It is really the version of the controller that extends the JamCtrlMgr to interface with the HW.

Where JamCtrlSim extends JamCtrlMgr to interface with the simulator code.